docs/en/api/metrics-query-expression.md

```diff
@@ -206,3 +206,22 @@ aggregate_labels(total_commands_rate, SUM)
 
 ### Result Type
 The ExpressionResultType of the aggregateLabels operation is TIME_SERIES_VALUES.
+
+## ViewAsSequence Operation
+ViewAsSequence operation searches for expressions in the given list from start to end and returns the expression that not empty.
```

Comment:
I'm curious what does it mean `viewAsSequence`, the name doesn't look any relevant to the logic behind it: filter first non empty expression.

Reply:
short-circuit in logical operators means, if the left expression already satisfies the condition, the right expressions won't be evaluated at all, e.g., `list != null && list.isEmpty()`, if list is `null`, then the `list.isEmpty()` won't be evaluated at all, thus no NullPointerException, but from what you said 

> The return is not as soon as the first not-empty is found. I believe the actual execution is getting all expression results first and then check according to the seq.

IT's totally opposite to the case, so I think we should say the new function is not short-circuit, all expressions are evaluated no matter how